flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/rules/logical/FlinkSubQueryRemoveRule.scala

```diff
@@ -216,6 +216,18 @@ class FlinkSubQueryRemoveRule(
 
       // EXISTS and NOT EXISTS
       case SqlKind.EXISTS =>
+        // If the sub-query is guaranteed to produce at least one row, just return TRUE.
+        // If the sub-query is guaranteed to produce no rows then return FALSE.
```

Comment:
NIT, can we move the comment to it's nearest code snippet, e.g.:
```java
// If the sub-query is guaranteed to produce at least one row, just return TRUE.
if (minRowCount != null && minRowCount >= 1d) {
          return Option.apply(relBuilder.literal(!withNot))
        }

// If the sub-query is guaranteed to produce no rows then return FALSE.
if (maxRowCount != null && maxRowCount < 1d) {
          return Option.apply(relBuilder.literal(withNot))
        }
```

Reply:
oops, seems I missed, it
makes, sense, thanks
done